src/backend/base/langflow/components/brightdata/brightdata_search_engine.py

```
@@ -0,0 +1,140 @@
+            info="Choose which search engine to use for the search",
+            placeholder="google, bing, yandex",
+        ),
+
+    ]
+
+    outputs = [
+        Output(display_name="Search Results", name="results", method="search_web"),
+    ]
+
+    def _build_search_url(self, engine: str, query: str) -> str:
+        """Build the search URL for the specified engine"""
```

Ruff (0.11.9)

First line should end with a period, question mark, or exclamation point

Add closing punctuation

(D415)